### ISSUE TITLE
Lambda-Promtail S3 Event from another account return error

### ISSUE THREAD

**Issue by svenikea** [grafana/loki#10199]
Hi guys, I set up my lambda promtail in Account A to read the data pipeline log from S3 bucket in Account B through an S3 Event notification but the lambda returns an error. 

Here is the error
```
2023/08/09 09:04:09 calling the handler function resulted in a panic, the process should exit
Unknown application error occurred
Runtime.Unknown
```
The lambda from account A has permission to retrieve the S3 object from account B

**Comment by svenikea** [comment]
Maybe I was sending data pipeline log from S3 which was not supported at the moment?

**Comment by cstyan** [comment]
It seems like there should be more logging here if lambda-promtail ran. Unfortunately there's not much we can do/suggest without more information.

**Comment by svenikea** [comment]
Yeah, at the moment, I've noticed that lambda-promtail only supports ALB logs from an S3 bucket. So, I've set up a custom lambda function to meet my requirements.

I will close this ticket now. Thank you for your support
